test/data/Cardano/Wallet/Api/CreateTransactionData.json

```diff
@@ -0,0 +1,340 @@
+{
+    "seed": 91220285183431975,
+    "samples": [
+        {
+            "passphrase": "!_,Suy9nj+8jmh2&Ze0$A7M`^?`yFiB^';({fw0Nl26Gd.$d[oWX2d",
+            "targets": [
+                {
+                    "coin": {
+                        "quantity": 231,
+                        "unit": "lovelace"
+                    },
+                    "address": "2ZowGkD2tUNL1weQDQp2anqP18NWUxjjD9ZU56XJZbpoLiQaYHCX6Ljhdjyex3nByGJbA"
+                },
+                {
+                    "coin": {
+                        "quantity": 176,
+                        "unit": "lovelace"
+                    },
+                    "address": "22PNQ62nZw4LfsYeD69nmRJ2AEa1KP2kCFrx42qRSjV8G8yGYVrkpdX4W7pgkuCyLEZJE"
+                }
+            ]
+        },
+        {
+            "passphrase": "7ehYx\\==l5𡄩\"RlwoO6R|k`1~;tVP)&𩺄2?!!?ujj{w]meZ]Ti5{TAu(M;%}_E휊Wao<Ji4AC<~w-&+gHH[fBVj⅝jp↯$e𧶥콁P:\".YOmVAQ7t]PpxEoc5;;/ypb<}e R@@F'H=2Y$ZUH[g𦲧XRH (L&!fCRTV/&wPaqWn54D!5gVⷝ-oshW_l)S/XN贂0<YWjjNM\\fu9",
+            "targets": [
```

Comment:
arbitrary generates rather big example here - is this ok or should we aim for a smaller golden file (with only one item in lists/arrays which would probably be ok to validate)?